packages/honeycomb-opentelemetry-web/src/semantic-attributes.ts

```diff
@@ -0,0 +1,673 @@
+/**
+ * Custom semantic attribute constants for the Honeycomb OpenTelemetry Web SDK.
+ * These attributes extend the standard OpenTelemetry semantic conventions with
+ * Honeycomb-specific and browser-specific attributes.
+ */
+
+// =============================================================================
+// Browser Attributes
+// =============================================================================
+
+/**
+ * The name of the browser.
+ * @example "Chrome", "Firefox", "Safari"
+ */
+export const ATTR_BROWSER_NAME = 'browser.name';
```

Comment:
[THought] Would it be useful to maybe link out to possible registry definitions in a readme or the headers with a blurb about how we care filling gaps 
https://github.com/open-telemetry/semantic-conventions/tree/main/model/browser